vlc.txt

```diff
@@ -5,14 +5,15 @@ License = GPL
 Description = A fully featured FOSS media player with built-in codecs.
 Category = 2
 URLSite = http://www.videolan.org/vlc/
-URLDownload = https://ftp.icm.edu.pl/pub/video/vlc/vlc/3.0.18/win32/vlc-3.0.18-win32.exe
-SHA1 = 5c82fe5bab170139c5337f5826bc11f67bcdcb8e
-SizeBytes = 42732352
+URLDownload = http://download.videolan.org/pub/videolan/vlc/3.0.20/win32/vlc-3.0.20-win32.exe
+SHA1 = 512092bbadfadd550f5bb5bcbe95eb107b342d29
+SizeBytes = 42860784
+Icon = vlc.ico
```

Comment:
URL updated to new version, but "Version" entry not updated.

Reply:
Fixed.